### PR TITLE
Upgrade to node 6 & 8 for drone.yml

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
The aim is to fix the tests on drone
and also use the latest versions

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 pipeline:
-  build_4:
-    image: node:4.4.2
+  build_6:
+    image: node:6
     commands:
-      - npm --loglevel warn install -g npm@3
+      - npm --loglevel warn install
```
Comment (sulthan-ahmed) [inline .drone.yml]: removed the install of npm 3 because it doesn't seem relevant to node 6

Comment (lennym) [reply .drone.yml]: That line needs to be removed. Now you have `npm --loglevel warn install` running twice.

**Comment by sulthan-ahmed** [comment]
I need add a drone signature, I'll do that now
